Dockerfile.run

```diff
@@ -1,17 +1,11 @@
 FROM heroku/heroku:18
 
-RUN ln -s /workspace /app
-
 ARG pack_uid=1000
 ARG pack_gid=1000
-ENV CNB_USER_ID=${pack_uid}
-ENV CNB_GROUP_ID=${pack_gid}
 
-RUN mkdir -p /workspace
-RUN groupadd pack --gid ${pack_gid} && \
-    useradd -u ${pack_uid} -g ${pack_gid} -ms /bin/bash -d /workspace heroku
+RUN groupadd heroku --gid ${pack_gid} && \
+  useradd heroku -u ${pack_uid} -g ${pack_gid} -s /bin/bash -md /workspace && \
+  ln -s /workspace /app
```

Comment:
I think we can simplify this and remove `/app` from this and the builder. We don't expect to seamlessly support buildpacks and they will break if they don't support symlinks right?

Reply:
Oh, I see we had it before. I think my question still stands..but I'm not sure. We may be intending to support existing buildpacks more than I imagined.

Reply:
I think the original intent was to support legacy buildpacks via the shim. So, sort of seamless maybe? This particular line is only needed for the legacy node buildpack, which seems to hardcode `/app` into $PATH in it's .profile.d.